kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -63,6 +65,9 @@
   public static final String REGION_CONFIG = "s3.region";
   public static final String REGION_DEFAULT = Regions.DEFAULT_REGION.getName();
 
+  public static final String CANNED_ACL_CONFIG = "s3.canned.acl";
+  public static final String CANNED_ACL_DEFAULT = "Private";
```

Comment:
We should compare against the `toString` representation of the enum (`cannedAclHeader` internally) not the enum name. So, `private` and `bucket-owner-read` instead of `Private` and `BucketOwnerRead`

Reply:
Also, I'm thinking that for complete backwards compatibility, maybe the default should be `null` here, because that's what it is so far.